spring-ai-alibaba-core/src/main/java/com/alibaba/cloud/ai/advisor/DocumentRetrievalAdvisor.java

```diff
@@ -1,8 +1,19 @@
 /*
- * All rights Reserved, Designed By Alibaba Group Inc.
- * Copyright: Copyright(C) 1999-2024
- * Company  : Alibaba Group Inc.
+ * Copyright 2023-2024 the original author or authors.
```

Comment:
The code first started in 2024, maybe it is more appropriate to change it to 2024